.github/workflows/sync-to-oss.yml

```
@@ -0,0 +1,314 @@
+            echo "🔧 Copying bin directory..."
+            cp -r bin /tmp/maria-oss/
+          fi
+          
+          # Copy scripts for post-install
+          if [ -d "scripts" ]; then
+            echo "📜 Copying scripts directory..."
+            mkdir -p /tmp/maria-oss/scripts
+            cp scripts/post-install.js /tmp/maria-oss/scripts/ 2>/dev/null || true
+            cp scripts/auto-start-llm.sh /tmp/maria-oss/scripts/ 2>/dev/null || true
+          fi
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)